### DISCUSSION TITLE
Error de memoria

### DISCUSSION THREAD

**Discussion by rocimarquez** [IIC2133-PUC/2024-2#35] [Tarea 0] [answered]
Hola, estaba probando el test de Part-2-easy y me tira error de memoria en una parte en que los test 1 no me tira error en una linea donde leo "K" con fscanf, como si estuviese llamando a algo que ya liberé. Son solo las 3 primeras funciones las que se ejecutan por lo que no veo porque funciona en la Part1 y no en la Part2. Dejo el output de valgrind aquí y mi repo ya está actualizado.
==2438== Invalid read of size 4
==2438==    at 0x48C4A72: __vfscanf_internal (vfscanf-internal.c:345)
==2438==    by 0x48C421C: __isoc99_fscanf (isoc99_fscanf.c:30)
==2438==    by 0x109905: main (main.c:150)
==2438==  Address 0xc0 is not stack'd, malloc'd or (recently) free'd
==2438==
==2438==
==2438== Process terminating with default action of signal 11 (SIGSEGV)
==2438==  Access not within mapped region at address 0xC0
==2438==    at 0x48C4A72: __vfscanf_internal (vfscanf-internal.c:345)
==2438==    by 0x48C421C: __isoc99_fscanf (isoc99_fscanf.c:30)
==2438==    by 0x109905: main (main.c:150)
==2438==  If you believe this happened as a result of a stack
==2438==  overflow in your program's main thread (unlikely but
==2438==  possible), you can try to increase the size of the
==2438==  main thread stack using the --main-stacksize= flag.
==2438==  The main thread stack size used in this run was 8388608.
==2438==
==2438== HEAP SUMMARY:
==2438==     in use at exit: 0 bytes in 0 blocks
==2438==   total heap usage: 2 allocs, 2 frees, 944 bytes allocated
==2438==
==2438== All heap blocks were freed -- no leaks are possible

**Comment by CataAlamos** [comment, marked as answer]
Hola! Mira la issue #28.
